README.md

```diff
@@ -1 +1,49 @@
-# My-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Desktop version Personal Portfolio and deploy to online
+
+> This is a project for Desktop Version portfolio that deploy on online using github pages.
```

Comment:
[OPTIONAL] A better description could be:

> This website describes my Professional Experience, Achievements, and Personal Projects.

Since this website will be your portfolio and you will make it responsive in the upcoming weeks for both Mobile and Desktop 🤗